src/main/java/org/verdictdb/core/querying/ola/AsyncQueryExecutionPlan.java

```diff
@@ -527,120 +511,160 @@ void addTierColumnsRecursively(
       List<SelectItem> newSelectlist =
           createUnfoldSelectlistWithBasicAgg(
               ((AggExecutionNode) node).getSelectQuery(), node.getAggMeta());
-      
+
       List<ProjectionNode> projectionNodeList = new ArrayList<>();
       for (ExecutableNodeBase source : node.getSources()) {
         if (source instanceof ProjectionNode && block.getNodesInBlock().contains(source)) {
           projectionNodeList.add((ProjectionNode) source);
         }
       }
-      Map<ScrambleMeta, String> scrambleMetaAndTierColumnAlias = addTierColumnToSelectListAndGroupBy(
-          ((AggExecutionNode) node).getSelectQuery(),
-          newSelectlist,
-          scrambleMeta,
-          projectionNodeList);
+      Map<ScrambleMeta, String> scrambleMetaAndTierColumnAlias =
+          addTierColumnToSelectListAndGroupBy(
+              ((AggExecutionNode) node).getSelectQuery(),
+              newSelectlist,
+              scrambleMeta,
+              projectionNodeList);
       ((AggExecutionNode) node).getSelectQuery().clearSelectList();
       ((AggExecutionNode) node).getSelectQuery().getSelectList().addAll(newSelectlist);
       aggColumnIdentiferNum = 0;
-      
+
       node.getAggMeta().setTierColumnForScramble(scrambleMetaAndTierColumnAlias);
-      
+
     } else if (node instanceof ProjectionNode) {
       rewriteProjectionNodeToAddTierColumn(block, (ProjectionNode) node);
     }
   }
-  
+
   /**
-   * For example, convert
-   * 1. avg(price)            -> sum(price) as 'agg0', count(price) as 'agg1'
-   * 2. sum(price) / count(*) -> sum(price) as 'agg0', count(*) as 'agg1'
+   * For example, convert 1. avg(price) -> sum(price) as 'agg0', count(price) as 'agg1' 2.
```

Comment:
I more like the original comment... I guess this is due to the auto indent thing.